### ISSUE TITLE
Add visualization for training and evaluation metrics

### ISSUE THREAD

**Issue by cedrickchee** [cedrickchee/capsule-net-pytorch#1]
Plots:
- Training loss and accuracy
- Test/validation accuracy

The plan is to use evaluate which of these libraries to use for data visualization:
- [TensorBoard for PyTorch](https://github.com/lanpa/tensorboard-pytorch)
- [Visdom](https://github.com/facebookresearch/visdom)

I don't have experience of using Visdom. I have used TensorBoard for TensorFlow before.

**Comment by cedrickchee** [comment]
Implemented using TensorBoard in this commit 66b65e044c19680cb76a31dad6cc983b4aed6375.
